torch/distributed/rpc/api.py

```diff
@@ -520,6 +520,13 @@ def rpc_sync(to, func, args=None, kwargs=None):
         args (tuple): the argument tuple for the ``func`` invocation.
         kwargs (dict): is a dictionary of keyword arguments for the ``func``
                        invocation.
+        timeout (float, optional): timeout to use for this RPC. If the RPC
```

Comment:
we need to say what is the time unit for this timeout.

Reply:
Added